docs/cloud/dbaas/operations.md

```diff
@@ -35,6 +35,26 @@ The commands to use:
 
 5. Verify with your preferred tool that your database is working as expected.
 
+## Major database upgrades
+Major version database upgrades works from the users point the same way but in the background
+happens more stuff, and is therefor more riskily.
+
+Things that you need to take into account when doing a major database version upgrade:
+1. You have a recent enough backup that you can use if the upgrade fails.
+2. You have tested doing the upgrade on database restored from the backup.
+3. You reserve plenty of time for the upgrade.
+4. You have considered if you benefit of using a bigger flavor while doing the upgrade.
```

Comment:
You have considered if you would benefit from using a bigger flavor while doing the upgrade.